docs/eventing/getting-started.md

```diff
@@ -250,22 +253,25 @@ EOF
 ## Sending events to the broker
 
 1. SSH into the pod by running the following command:
-    ```
+
+    ```shell
     kubectl -n event-example attach curl -it
     ```
 
     You will see a prompt similar to the following:
-    ```
+
+    ```shell
     Defaulting container name to curl.
     Use 'kubectl describe pod/ -n event-example' to see all of the containers in this pod.
     If you don't see a command prompt, try pressing enter.
     [ root@curl:/ ]$
     ```
 
 1. Make a HTTP request to the broker. To show the various types of events you can send, you will make three requests:
-    - To make the first request, which creates an event that has the `type`
-       `greeting`, run the following in the SSH terminal:
-       ```
+
+    - To make the first request, which creates an event that has the `type` `greeting`, run the following in the SSH terminal:
+
+       ```shell
```

Comment:
that's nice to see the curl example in the getting started

Reply:
I just cleaned things up, so credit goes to whoever wrote it initially 😄